nav2_controller/src/nav2_controller.cpp

```diff
@@ -41,6 +44,12 @@ ControllerServer::ControllerServer()
   declare_parameter("controller_frequency", 20.0);
 
   declare_parameter("controller_plugins", default_ids_);
+  declare_parameter(
```

Comment:
We have this new method or getting plugin names: https://github.com/ros-planning/navigation2/blob/f3b7be2397f585c53d4940b0fc516d8fe4738670/nav2_util/include/nav2_util/node_utils.hpp (examples https://github.com/ros-planning/navigation2/pull/1808)

For this case, we want `progress_checker_plugin` and `goal_checker_plugin` to look in those namespaces for the `plugin` field. We can also use those namespaces for the parameters too if we want and shuffle it around.

Reply:
Done.